test/unit/aggregate_store_test.rb

```diff
@@ -25,6 +25,17 @@ def add(attr, message)
       @store.aggregate_attribute(:name, :string)
     end
 
+    should "pass aggregate_db_storage_type option to all attribute handlers if aggregate_db_storage_type is not nil" do
+      store = Class.new { }
+      store.send(:include, Aggregate::AggregateStore)
+      store.define_singleton_method(:aggregate_db_storage_type) { :elasticsearch }
```

Comment:
Can we create a dummy class outside of the tests, instead of building it separately multiple times?